mrjones2014/yaclt
# Add middleware to check for options not known to any command

Resolves: #155 

## How to Test

1. Run `yarn build`
2. Run `./dist/index.js validate` -- it should succeed
3. Run `./dist/index.js validate --test` -- it should give error about unknown argument
